ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -507,9 +514,11 @@ impl<'a> Assemble<'a> {
                     continue;
                 }
                 jit_ir::Inst::Guard(i) => self.cg_guard(iidx, i),
-                jit_ir::Inst::TraceLoopStart => self.cg_traceloopstart(),
-                jit_ir::Inst::TraceLoopJump => self.cg_traceloopjump(),
-                jit_ir::Inst::RootJump => self.cg_rootjump(self.m.root_jump_addr()),
+                jit_ir::Inst::TraceHeaderStart => self.cg_header_start(),
+                jit_ir::Inst::TraceHeaderEnd => self.cg_header_end(),
+                jit_ir::Inst::TraceBodyStart => self.cg_body_start(),
+                jit_ir::Inst::TraceBodyEnd => self.cg_body_end(iidx),
+                jit_ir::Inst::RootJump => self.cg_rootjump(iidx, self.m.root_jump_addr()),
```

Comment:
For consistency should this be `SideTraceEnd` or similar?

Reply:
Could do.

Reply:
[e74e498](https://github.com/ykjit/yk/pull/1510/commits/e74e4983fd39ab895860b92ee3541244233e45ea)